conduct/conduct_enforcement.md

```diff
@@ -0,0 +1,118 @@
+and fair. It should be considered an internal document, but we're publishing it
+publicly in the interests of transparency.
+
+
+## The Code of Conduct Committee
+
+All responses to reports of conduct violations will be managed by a Code of
+Conduct Committee ("the committee").
+
+The Jupyter Steering Council will establish this committee, comprised of at
+least five members. One member will be designated chair of the group and will be
+responsible for all reports back to the Steering Council. The Steering Council
+will review membership on a regular basis.
+
+## How the committee will respond to reports
+
+When a report is sent to the committee they will immediately reply to the report
+to confirm receipt. This reply must be sent within 24 hours, and the group
+should strive to respond much quicker than that.
+
+See the [Reporting Guidelines](*conduct_reporting_online.md*) for details of
+what reports should contain. If a report doesn't contain enough information, the
+committee will obtain all relevant data before acting. The committee is
+empowered to act on the Steering Council’s behalf in contacting any individuals
+involved to get a more complete account of events.
+
+The committee will then review the incident and determine, to the best of their
+ability:
+
+* What happened
+* Whether this event constitutes a Code of Conduct violation
+* Who, if anyone, was responsible for the violation
+* Whether this is an ongoing situation, and there is a threat to anyone's
+  physical safety
+
+This information will be collected in writing, and whenever possible the group's
+deliberations will be recorded and retained (i.e. chat transcripts, email
+discussions, recorded voice conversations, etc).
+
+It is important to retain an archive of all activities of this committe to
+ensure consistency in behavior and provide institutional memory for the
+project.
+
+The Code of Conduct Committee should aim to have a resolution agreed upon within
+one week. In the event that a resolution can't be determined in that time, the
+committee will respond to the reporter(s) with an update and projected timeline
+for resolution.
+
+
+## Acting Unilaterally
+
+If the act is ongoing, or involves a threat to anyone's safety (e.g. threats of
+violence), any committee member may act immediately (before reaching consensus)
+to address the situation. In ongoing situations, any member may decide to employ
+any of the tools available to the committee, including bans and blocks.
+
+If the incident involves physical danger, any member of the committee may -- and
+should -- act unilaterally to protect the safety of those involved. This can
+include contacting law enforcement (or other local personnel) and speaking on
+behalf of the Jupyter Steering Council.
+
+In situations where an individual committee member acts unilaterally, they must
+report their actions to the committee for review within 24 hours.
+
```

Comment:
Do we have a policy on Jupyter events where a CoC committee member isn't present. We should probably formalize the idea of a delegate who acts on behalf of the CoC team at specific events.